dcos_test_utils/dcos_cli.py

```diff
@@ -18,7 +18,7 @@
 
 log = logging.getLogger(__name__)
 
-DCOS_CLI_URL = os.getenv('DCOS_CLI_URL', 'https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.12/dcos')
+DCOS_CLI_URL = os.getenv('DCOS_CLI_URL', 'https://downloads.dcos.io/cli/releases/binaries/dcos/linux/x86-64/latest/dcos')  # noqa: E501
```

Comment:
Optional - Rather than `noqa` perhaps we could have `_DEFAULT_CLI_URL = ''https://downloads.dcos.io/cli/releases/binaries/dcos/linux/x86-64/latest/dcos''` on the line above.